GsaGH/Components/0_Model/OpenModel.cs

```diff
@@ -38,8 +38,8 @@ public class OpenModel : GH_OasysDropDownComponent {
 
     public void OpenFile() {
       var fdi = new OpenFileDialog {
-        Filter = "GSA Files(*.gwb)|*.gwb|All files (*.*)|*.*",
-      }; //"GSA Files(*.gwa; *.gwb)|*.gwa;*.gwb|All files (*.*)|*.*"
+        Filter = "GSA Files(*.gwa; *.gwb)|*.gwa;*.gwb|All files (*.*)|*.*",
```

Comment:
can you remove the space here pls:
`Filter = "GSA Files(*.gwa;*.gwb)|*.gwa;*.gwb|All files (*.*)|*.*",`

Reply:
ok